README.md

```diff
@@ -24,51 +24,45 @@ If you build `gLike` using an older ROOT version, the interface to FITS data for
-This is the `gLike` directory we downloaded from git
-```shell
-cmake /path/to/the/source/gLike/dir/
-```
-if you want to unlock FITSIO support, use the cache variable `USE_FITSIO` (can be set to `ON` or `True`, default is `OFF`)
-```shell
-cmake /path/to/the/source/gLike/dir/ -DUSE_FITSIO=ON
-```
-at this point `cmake` will have automatically generated the `Makefile` for us, so we just
-```shell
-make
-```
-and install
-```shell
-make install
-```
-this last step will make gLike libraries available system-wise, i.e. libraries will be saved in `/usr/local/lib` and executables in `/usr/local/bin`.
+
+3. There are two available installation modes, the first with an already provided `Makefile`, the second with `cmake`.    
+**Note**: the isntallation with the `Makefile` will be deprecated in the next `gLike` release
+
+4. `Makefile` installation
+    - Define the environment variable GLIKESYS pointing to your main gLike directory, e.g., for bash:    
+    `export GLIKESYS="/Users/rico/gLike/"`    
+    and include it in the library path, eg. for Mac:    
+    `export DYLD_LIBRARY_PATH="${GLIKESYS}:${DYLD_LIBRARY_PATH}"`;
+    - Create the gLike library by typing `make` in the main gLike directory;
+    -  [Optional] Create the gLike online documentation with `make doc`. 
+
+5. `cmake` installation
+    - Define the environment variable `GLIKESYS` pointing to the directory _where you want to build_ gLike,    
+      you can create the build directory within the original `gLike` repository itself    
+      `cd gLike`   
+      `mkdir build`     
+      or outside in another repository    
+      `mkdir ~/gLike_build`.    
+      Just export this build directory as an environment variable in your `.bashrc`:     
+      `export $GLIKESYS="/home/user/gLike/build`    
+      or    
+      `export $GLIKESYS="/home/user/gLike_build`   
+      if the build directory is outside the one containing the source code.   
+
+    - go in the build directory      
+      `cd $GLIKESYS`   
+    - run `cmake`, the first argument being the directory in which the `CMakeLists.txt` for building the project is defined 
+      (the `gLike` directory we downloaded from git)     
+      `cmake /path/to/the/source/gLike/dir/`    
+      if you want to unlock FITSIO support, use the cache variable `USE_FITSIO` (can be set to `ON` or `True`, default is `OFF`)    
+      `cmake /path/to/the/source/gLike/dir/ -DUSE_FITSIO=ON`    
+    - at this point `cmake` will have automatically generated the `Makefile` for us, so we just    
+      `make`         
+      and install    
+      `make install`    
+      this last step will make gLike libraries available system-wise, i.e. libraries will be saved in `/usr/local/lib` and executables in `/usr/local/bin`.
 
 ### load gLike libraries in ROOT
 If you want to load the gLike libraries each time you open ROOT, modify (or create) in your home a `.rootrc` file, inserting (editing) the line
```

Comment:
I suggest to have this .rootrc with the call to rootlogon.C included in the $GLIKESYS directory

Reply:
It's already there!

Reply:
I added a line in the docs, specifying the example `.rootrc`is in the main directory.